<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" colspan="3">
    <widget class="QComboBox" name="ParamList">
     <item>
      <property name="text">
       <string>Mobility of holes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mobility of electrons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Electrical conductivity</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="Start"/>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>for</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="End"/>
   </item>
   <item row="2" column="5">
    <widget class="QLineEdit" name="Points"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_2">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>            to</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="PlotBtn">
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
